language/move-model/src/model.rs

```diff
@@ -2351,10 +2351,12 @@ impl<'env> FunctionEnv<'env> {
 
     /// Returns true if this function is public.
     pub fn is_public(&self) -> bool {
-        let view = self.definition_view();
```

Comment:
@wrwg would you mind checking whether this change is OK to you and also suggest how we should further propagating the visibility information in the `move-model`? For example, shall we change the function `pub fn is_public(&self) -> bool` into `pub fn visibility(&self) -> Visibility`?

Reply:
I suggest to move forward with keeping this API as is and just adjust the impl as you do. We (prover team) can then come back and change the API once we use this new info.